### PR TITLE
Merging Feat 2 into Dev

### PR DESCRIPTION
**Pull request by Reece-elder**
Pls approve this its important code 🌟

### CODE REVIEW

File: feat_2.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+print("Hello World")
```
Comment (Samelamename) [inline feat_2.py:1]: Alls good

**Review by Samelamename — CHANGES_REQUESTED** [review]
No!

File: feat_2.py (hunk 1 above)
Comment (Reece-elder) [reply feat_2.py:1]: Okay, all fixed! 😄

**Review by Samelamename — APPROVED** [review]
Cool

**Pull request merged**